core-java-modules/core-java-string-operations-8/src/test/java/com/baeldung/countupperandlowercasechars/CountUpperAndLowercaseCharsUnitTest.java

```diff
@@ -0,0 +1,85 @@
+package com.baeldung.countupperandlowercasechars;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class CountUpperAndLowercaseCharsUnitTest {
+    private static final String MY_STRING = "Hi, Welcome to Baeldung! Let's count letters!";
+    private static final LetterCount EXPECTED = new LetterCount(4, 31);
+
+    @Test
+    void whenIteratingCharArrayCompareAndCount_thenGetExpectedResult() {
+        int upperCnt = 0;
```

Comment:
Can you rename the `upperCnt` and `lowerCnt` variables to `upperCount` and `lowerCount`, or just `upper` and `lower`, and keep the naming consistent throughout the tests and the `LetterCount` fields too

(the letters cnt sound too similar to a curse word, and it's generally more helpful to a reader to see whole words than abbreviations)

Reply:
Definitely! Thank you for the suggestion.
renamed in the new commit